src/components/Layout.tsx

```diff
@@ -46,15 +46,15 @@ export function Layout() {
 									setActiveChar(character.id)
 								}}
 							>
-								<div className="char-button">
+								<div className="char-button" id="character">
```

Comment:
Technically, in HTML ids should be unique on the page so these should be classNames

Reply:
Oh, right. Not a frontend dev :sweat:. I'll do a second pass a bit later.

Reply:
No problem!